queue/public.php

```diff
@@ -22,14 +22,12 @@
 	<!-- Body Section with list of clients -->
 	<div class="container-fluid queue" ng-if="appointments.length > 0" ng-cloak>
 		<div class="row queue-header py-2 bg-secondary text-light font-weight-bold">
-			<div class="col col-1 queue-id">Id.</div> <!-- TODO: Remove ID from here -->
-			<div class="col col-3 queue-name">Name</div>
+			<div class="col col-4 queue-name">Name</div>
 			<div class="col col-5 queue-progress">Progress</div>
 			<div class="col col-3 queue-time">Time</div>
 		</div>
 		<div class="row queue-row" ng-repeat="appointment in appointments | orderBy:'scheduledTime'" ng-if="appointment.completed == null" ng-class-even="'bg-light'">
```

Comment:
I would encourage breaking up lines like this with many different attributes, like you did below, it helps with readability.